docker-compose.yml

```diff
@@ -0,0 +1,99 @@
+mamangcontrol:
+  restart: always
+  build: ../mama-ng-control
```

Comment:
Are these relative directories correct? This file is in the root, and the Dockerfiles are in `docker-vms/...`